recipes/karchive/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+from conan.tools.files import rename
+from conans import ConanFile, CMake, tools
+import shutil
+
+
```

Comment:
Please always add this attribute

```suggestion
required_conan_version = ">=1.33.0"


```